packages/lib/src/vis/shared/SelectionMesh.tsx

```diff
@@ -57,8 +58,11 @@ function SelectionMesh(props: Props) {
       (target as Element).releasePointerCapture(pointerId);
       setEndPoint(point);
       setDrag(false);
+      if (onSelection && startPoint && endPoint) {
+        onSelection(startPoint, endPoint);
+      }
```

Comment:
I thought it was best to run the callback after `onPointerUp` rather than use a `useEffect`.

Reply:
Yep, that's great 👍